Cpp/odin-views/google.cpp

```diff
@@ -52,7 +63,7 @@ namespace {
                     // Use access token as google ID
                     fostlib::insert(user_detail, "sub", access_token);
                     fostlib::insert(user_detail, "name", "Test User");
-                    fostlib::insert(user_detail, "email", access_token + "@mail.com");
+                    fostlib::insert(user_detail, "email", access_token + "@example.com");
```

Comment:
Why are we putting invalid email addresses into the database?

Reply:
This one is use for mock testing only

Reply:
Oh OK then.